### PR TITLE
Update to PatternFly 4.4

### PR DESCRIPTION
**Pull request by rvsia**
BLOCKED BY 
- [x] https://github.com/patternfly/patternfly-react/issues/4383
- waiting for new release of PF4

TODO: 
- [x] test in the UI after Sources wizard beta is released
- [x] update tests

WAIT FOR:
- [x] final release of FC
- [x] final release of sources wizard
- [x] final release of DDF
- [x] final release of frontend utilities
- [x] final release of frontent notifications

### CODE REVIEW

**Comment by syncrou** [comment]
cc @syncrou

**Comment by rvsia** [comment]
@Hyperkid123 All issues I found are fixed, this is ready to be reviewed.

File: src/components/SourceEditForm/SourceEditModal.js [hunk 1]
```diff
@@ -66,13 +66,13 @@ const SourceEditModal = () => {
     if (isLoading) {
         return (
             <Modal
-                title={intl.formatMessage({
+                aria-label={intl.formatMessage({
```
Comment (Hyperkid123) [inline src/components/SourceEditForm/SourceEditModal.js:69]: Is the title no longer required?

File: src/components/SourceEditForm/SourceEditModal.js [hunk 2]
```diff
@@ -87,13 +87,13 @@ const SourceEditModal = () => {
             <sourceEditContext.Provider value={{ setState, source, editing }}>
                 {state.isAuthRemoving && <RemoveAuth {...state.isAuthRemoving}/>}
                 <Modal
-                    title={intl.formatMessage({
+                    aria-label={intl.formatMessage({
```
Comment (Hyperkid123) [inline src/components/SourceEditForm/SourceEditModal.js:90]: ^^

File: src/components/SourceEditForm/SourceEditModal.js (hunk 1 above)
Comment (rvsia) [reply src/components/SourceEditForm/SourceEditModal.js:69]: Yep. Do not use title when there is a custom header. Before that, the title was just used to generate aria-label, so now you have to pass it directly.

Comment (Hyperkid123) [reply src/components/SourceEditForm/SourceEditModal.js:69]: Ah custom header :+1:

**Comment by syncrou** [comment]
07/02/2020 Merging to CI-BETA & QA-BETA ( via new custom_release script per the outcome of this slide (2) https://docs.google.com/presentation/d/1a-sf-kK4_g8SUiEQSky-khQVAOr2FzkFm9UzWxOmaMM/edit#slide=id.g8a08c2b83d_0_28

**Pull request merged**